ci-operator/step-registry/baremetal/lab/upi/conf/disconnected/baremetal-lab-upi-conf-disconnected-chain.yaml

```diff
@@ -1,20 +1,20 @@
 chain:
-  as: baremetal-lab-upi-conf-disconnected-proxy
+  as: baremetal-lab-upi-conf-disconnected
   steps:
     - ref: baremetal-lab-pre-reserve-nodes
     - ref: baremetal-lab-pre-dhcp-conf
     - chain: baremetal-lab-pre-load-balancer
     - ref: baremetal-lab-pre-dns
     - ref: baremetal-lab-pre-firewall
     - ref: ipi-install-rbac
-    - ref: mirror-images-by-oc-adm
+    - ref: mirror-images-by-oc-adm-in-bastion
+    - ref: mirror-images-qe-test-images
```

Comment:
I am not sure if the mirror server in the baremetal environment configured proxied registry (refer to `ignition-bastionhost`) on 6001 and 6002, if it is already configured, then we do need `mirror-images-qe-test-images`, it will mirror all the est images, that would consume disk space a lot.

Reply:
bastion host has registry configured on 6001 and 6002. I'll test without the `mirror-images-qe-test-images` step

Reply:
Works without `mirror-images-qe-test-images`